vignettes/join-keys.Rmd

```diff
@@ -8,39 +8,98 @@ vignette: >
   %\VignetteEncoding{UTF-8}
 ---
 
+```{r, include=FALSE}
+devtools::load_all()
+```
+
 ## Overview
 
-`teal.data` gives the possibility to define individual keys per dataset and to define the relations to other datasets.
+`teal.data` gives the possibility to define primary keys of a dataset and to define the relations to other datasets.
 
-Each table can have a set of keys that inform its structure and relation to other tables by specifying:
+Each dataset can have a set of keys that inform its structure and relation to other tables by specifying:
 
-- columns consisting the primary key
-- merge keys, a concept similar to `SQL`'s foreign key.
+- Columns consisting of the primary key
+- Merge keys, a concept similar to `SQL`'s foreign key.
 
-Usually, an application developer needs to specify the keys manually, but in case of datasets named according to the `ADaM` standard, `teal` can assign the keys on its own. 
+Usually, an application developer needs to specify the keys manually, but in case of datasets named according to the `ADaM` standard, `teal` can assign the keys on its own.
```

Comment:
This is not true and vignette `including-adam-data-in-teal` shouldn't be mentioned. CDISC keys are handled in this package by relevant default (default_cdisc_join_keys and cdisc_data)

Reply:
I was considering changing the documentation on that vignette, but that was wrong as the `default_cdisc_join_keys` lives here.

I've added a new section at the bottom. Please take a look at it.